static/sass/components/_forms.scss

```diff
@@ -0,0 +1,216 @@
+%input_text {
+  @extend .font-sansserif;
+  border: 1px solid $color-gray-light;
+  height: 27px;
```

Comment:
I try to avoid magic pixel numbers (but it's not always possible and this may be an exception). Could the heights on this and `%input_text--l` be set as variables? Maybe we need an update to the `utils/_forms.scss` for input heights, and could standardize it in the main repo too with a variable.